copier/cli.py

```diff
@@ -120,7 +122,15 @@ class CopierApp(cli.Application):
         ["-n", "--pretend"], help="Run but do not make any changes"
     )
     force: cli.Flag = cli.Flag(
-        ["-f", "--force"], help="Overwrite files that already exist, without asking"
+        ["-f", "--force"],
+        help="Overwrite files that already exist, without asking. Use default answers to questions, which might be null if not specified.",
```

Comment:
To avoid repetition, I think I prefer:

```suggestion
        help="Same as `--default --overwrite`.",
```